pulumi/src/assets/main.go

```diff
@@ -6,11 +6,11 @@ import (
 	"github.com/tinkerbell/infrastructure/src/internal"
 )
 
-type AssetsDns struct {
+type DNS struct {
 	Cname *ns1.Record
 }
 
-func CreateAssetsDns(ctx *pulumi.Context, infrastructure internal.Infrastructure) (AssetsDns, error) {
+func CreateAssetsDNS(ctx *pulumi.Context, infrastructure internal.Infrastructure) (DNS, error) {
```

Comment:
I would expect this to be named `CreateDNS` which would be in line with `ManageDNS`

Reply:
Great idea. Done!